<!-- -->
<launch>
  <arg name="use_crosswalk_detection" default="true" />
  <arg name="points_topic" default="points_no_ground" />
  <arg name="velocity_offset" default="1.2" />
  <arg name="decelerate_vel_min" default="1.3" />
  <arg name="remove_points_upto" default="2.3" />
  <arg name="enable_multiple_crosswalk_detection" default="true" />
  <arg name="stop_distance_obstacle" default="10.0" />
  <arg name="stop_distance_stopline" default="5.0" />
  <arg name="detection_range" default="1.3" />
  <arg name="points_threshold" default="10" />
  <arg name="detection_height_top" default="0.2" />
  <arg name="detection_height_bottom" default="-1.7" />
  <arg name="deceleration_obstacle" default="0.8" />
  <arg name="deceleration_stopline" default="0.6" />
  <arg name="velocity_change_limit" default="9.972" />
  <arg name="deceleration_range" default="0" />
  <arg name="temporal_waypoints_size" default="100" />

  <node pkg="waypoint_planner" type="velocity_set_lanelet2" name="velocity_set" output="screen">
    <param name="use_crosswalk_detection" value="$(arg use_crosswalk_detection)" />
    <param name="enable_multiple_crosswalk_detection" value="$(arg enable_multiple_crosswalk_detection)" />
    <param name="points_topic" value="$(arg points_topic)" />
    <param name="velocity_offset" value="$(arg velocity_offset)" />
    <param name="decelerate_vel_min" value="$(arg decelerate_vel_min)" />
    <param name="remove_points_upto" value="$(arg remove_points_upto)" />
    <param name="stop_distance_obstacle" value="$(arg stop_distance_obstacle)" />
    <param name="stop_distance_stopline" value="$(arg stop_distance_stopline)" />
    <param name="detection_range" value="$(arg detection_range)" />
    <param name="points_threshold" value="$(arg points_threshold)" />
    <param name="detection_height_top" value="$(arg detection_height_top)" />
    <param name="detection_height_bottom" value="$(arg detection_height_bottom)" />
    <param name="deceleration_obstacle" value="$(arg deceleration_obstacle)" />
    <param name="deceleration_stopline" value="$(arg deceleration_stopline)" />
    <param name="velocity_change_limit" value="$(arg velocity_change_limit)" />
    <param name="deceleration_range" value="$(arg deceleration_range)" />
    <param name="temporal_waypoints_size" value="$(arg temporal_waypoints_size)" />
  </node>

</launch>
